Muthaias/global-explorer
# Feature/use vue to render UI

This branch replaces the old vanilla JS render logic with Vue.JS which improves both ergonomics and stability.

To test:
- Checkout and run
- Verify that actions are triggering navigation
- Verify that input actions can transfer data

This branch should not change the behavior of the application in any way.
